src/utils/ColorUtils.js

```diff
@@ -32,13 +32,23 @@ define(function (require, exports, module) {
     "use strict";
     
     /**
+     * Sorted array of all the color names in the CSS Color Module Level 3 (http://www.w3.org/TR/css3-color/)
+     * and "rebeccapurple" from CSS Color Module Level 4
```

Comment:
I don't think Module Level 4 is ready yet, while Level 3 is.
While it's still in development, we should state which names are available.